comp110-worksheet-7/DirectoryUtils.cs

```diff
@@ -24,37 +24,86 @@ public static bool IsDirectory(string path)
 		// Return the total size, in bytes, of all the files below the given directory
 		public static long GetTotalSize(string directory)
 		{
-			throw new NotImplementedException();
+            long sizeOfFiles;
+
+            string[] fileNum = Directory.GetFiles(directory, "*.*", SearchOption.AllDirectories);
+
+            foreach (string file in fileNum)
+            {
+                sizeOfFiles += GetFileSize(file);
```

Comment:
Error here "Use of unassigned local variable 'sizeOfFiles'" -- you need to initialise the variable to 0 when you first declare it